cli/cmd/options.go

```diff
@@ -622,6 +627,16 @@ func validateProxyValues(values *l5dcharts.Values) error {
 	return nil
 }
 
+func validatePolicy(policy string) error {
+	validPolicies := []string{"all-authenticated", "all-unauthenticated", "cluster-authenticated", "cluster-unauthenticated", "deny"}
+	for _, p := range validPolicies {
+		if p == policy {
+			return nil
+		}
+	}
+	return fmt.Errorf("--default-inbound-policy must be one of: %s", strings.Join(validPolicies, ", "))
```

Comment:
nit, take it or leave it: should we include the invalid value here?
```suggestion
	return fmt.Errorf("--default-inbound-policy must be one of: %s (got %s)", strings.Join(validPolicies, ", "), policy)
```